dietpi/dietpi-software

```diff
@@ -3617,6 +3642,16 @@ Package: openssl libssl*\nPin: origin packages.sury.org\nPin-Priority: -1' > /et
 
 		fi
 
+		software_id=125 # Synapse Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# APT package (have to install from backports until official repo supports ARM)
+			G_AGI -t buster-backports matrix-synapse
```

Comment:
best to my knowledge, buster-backports source list is not available on Raspberry OS 32bit by default.

Reply:
Well, I'll have to build it from source then.